cypress/integration/table_tool_spec.js

```diff
@@ -79,14 +79,27 @@ context('Table Tool Tile',function(){
     });
     describe('edit table entries', function(){
         it('will add content to table', function(){
-            tableToolTile.getTableCell().first().type('0');
-            tableToolTile.getTableCell().last().type('5{enter}');
-            tableToolTile.getTableCell().first().should('contain','0');
-            tableToolTile.getTableCell().eq(1).should('contain', '5');
-            tableToolTile.getTableRow().should('have.length',2);
+            tableToolTile.getTableCell().first().type('3');
+            tableToolTile.getTableCell().last().click();
+            cy.wait(100);
+            tableToolTile.getTableCell().first().should('contain','3');
+
+            tableToolTile.getTableCell().last().type('2');
+            tableToolTile.getTableCell().first().click();
+            cy.wait(100);
+            tableToolTile.getTableCell().eq(1).should('contain', '2');
+
+            tableToolTile.getTableCell().first().type('1');
+            tableToolTile.getTableCell().last().click();
+            cy.wait(100);
+            tableToolTile.getTableCell().first().should('contain','1');
```

Comment:
I must be missing something - I don't quite understand how we have 3 rows at the end of this test

Reply:
Edits in the input row result in new rows being created.

Reply:
Ah ok, it was not clear that this selector was only getting input rows, I thought it kept grabbing the first row.